steps/cli/oc.go

```diff
@@ -176,23 +176,22 @@ var _ = gauge.Step("Enable console plugin", func() {
 	oc.EnableConsolePlugin()
 })
 
-var _ = gauge.Step("Create quay secret for tekton chains", func(){
-	if os.Getenv("CHAINS_DOCKER_CONFIG_JSON") == "" || os.Getenv("CHAINS_CONFIG_JSON") == ""{
-		testsuit.T.Errorf("'CHAINS_DOCKER_CONFIG_JSON' and 'CHAINS_CONFIG_JSON' robot credentials system variables were not exported")
-	}else {
+var _ = gauge.Step("Create quay secret for Tekton Chains", func() {
+	if os.Getenv("CHAINS_DOCKER_CONFIG_JSON") == "" {
+		testsuit.T.Errorf("'CHAINS_DOCKER_CONFIG_JSON' robot credentials system variable is not exported")
+	} else {
 		dockerConfig := os.Getenv("CHAINS_DOCKER_CONFIG_JSON")
-		config := os.Getenv("CHAINS_CONFIG_JSON")
-		oc.CreateQuaySecretForTektonChains(dockerConfig, config)
+		oc.CreateQuaySecretForTektonChains(dockerConfig)
 	}
 })
 
-var _ = gauge.Step("Patch tekton config to sign and verify <resourceType> with Tekton Chains", func(resourceType string){
+var _ = gauge.Step("Modifying the TektonConfig for Tekton Chains by storage for storing signatures: <storage>", func(storage string) {
```

Comment:
It's a bit better but maybe we could do it in the same way what we did for addon tests or pruner tests
* https://github.com/openshift-pipelines/release-tests/blob/master/specs/operator/addon.spec#L15
* https://github.com/openshift-pipelines/release-tests/blob/master/specs/operator/auto-prune.spec#L28

If it would be too long, we could also split it into multiple tests, i.e. one would set taskrun format, another one taskrun storage etc